src/functions/accent.js

```diff
@@ -153,25 +162,25 @@ const htmlBuilder = (group, options) => {
     const accentWrap =
         buildCommon.makeSpan(["mord", "accent"], [accentBody], options);
 
-    if (supsubGroup) {
+    if (supSubGroup) {
         // Here, we replace the "base" child of the supsub with our newly
         // generated accent.
-        supsubGroup.children[0] = accentWrap;
+        supSubGroup.children[0] = accentWrap;
 
         // Since we don't rerun the height calculation after replacing the
         // accent, we manually recalculate height.
-        supsubGroup.height = Math.max(accentWrap.height, supsubGroup.height);
+        supSubGroup.height = Math.max(accentWrap.height, supSubGroup.height);
 
         // Accents should always be ords, even when their innards are not.
-        supsubGroup.classes[0] = "mord";
+        supSubGroup.classes[0] = "mord";
 
-        return supsubGroup;
+        return supSubGroup;
     } else {
         return accentWrap;
     }
 };
 
-const mathmlBuilder = (group, options) => {
+const mathmlBuilder: MathMLBuilder<"accent"> = (group, options) => {
```

Comment:
FYI: The reason for adding an explicit type in each of the `mathmlBuilder`s is to allow the type inference to succeed on `defineFunction` called later in this file.

Recall that `defineFunction` ties together the `type` argument with the output of `handler` and the input of `htmlBuilder` and `mathmlBuilder`. In this file, `htmlBuilder` is strictly more generic in this file (`HtmlBuilderSupSub` is a subtype of `HtmlBuilder` per the variance rules), but flow requires us to either *explicitly* cast it down from `HtmlBuilderSupSub` to `HtmlBuilder` to prevent type inference from failing, or, as it turns out, adding an explicit type for the `mathmlHandler` seems to be enough to satisfy it. IMO, none of this should matter since `defineFunction` explicitly specifies `type` which should be enough, to determine what the expected types of `handler`, `htmlBuilder`, and `mathmlBuilder` are, but it's still beyond flow's comfort zone.

Reply:
Isn't `HtmlBuilder` a subtype of `HtmlBuilderSupSub`?

Reply:
Functions are contravariant on their input types. Floor actually had some nice documentation on it: https://flow.org/en/docs/lang/variance/

Reply:
Note to self: as `HtmlBuilderSupSub` accepts less specific input, it's subtype of `HtmlBuilder`.